### ISSUE TITLE
将来会不会开放东南亚语音合成模型？

### ISSUE THREAD

**Issue by Pydataman** [alibaba-damo-academy/KAN-TTS#43]
谢谢

**Comment by GinChow** [comment]
Yes, we have released the Indonesian TTS model: https://modelscope.cn/models/damo/speech_sambert-hifigan_tts_indah_Indonesian_16k/summary, and Filipino is on the way.
